geopmpy/test/TestEndpoint.py

```diff
@@ -79,12 +76,12 @@ def test_reset_wait_loop(self):
         self._endpoint.reset_wait_loop()
 
     def test_endpoint_profile_name(self):
-        test_profile_name = 'my agent'
+        test_profile_name = 'my profile'
 
         def mock_profile_name(endpoint, name_max, name_cstr):
             for idx, char in enumerate(test_profile_name):
-                name_cstr[idx] = char.encode()
-            name_cstr[len(test_profile_name)] = b'\x00'
+                name_cstr[idx] = ord(char.encode())
+            name_cstr[len(test_profile_name)] = ord(b'\x00')
```

Comment:
Why was ord() necessary?

Reply:
@dannosliwcd may know better.   I think this is to properly emulate the character arrays returned by cffi when the interface sets string values (like profile name)